src/sparql/rm_dbxref_doid.ru

```diff
@@ -0,0 +1,20 @@
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
```

Comment:
There is afaics no need to split into two queries - just make 1 and run it for both ontologies. (rm_xref_by_prefix.ru) if a prefix should be deleted in one ontology it should be in all.